plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoPlugin.java

```diff
@@ -44,7 +44,7 @@ public void testCreateConnector()
         MongoPlugin plugin = new MongoPlugin();
 
         ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
-        Connector connector = factory.create("test", ImmutableMap.of("mongodb.seeds", seed), new TestingConnectorContext());
+        Connector connector = factory.create("test", ImmutableMap.of("mongodb.seeds", seed, "mongodb.connection-url", ""), new TestingConnectorContext());
```

Comment:
We shouldn't pass both seeds and connection-url.

Reply:
i have too. connection url cannot be null because of   requireNonNull(config, "config is null"); When null the below error occurs
Cannot invoke "String.isEmpty()" because the return value of "io.trino.plugin.mongodb.MongoClientConfig.getConnectionUrl()" is null
  at MongoClientModule.createMongoSession(MongoClientModule.java:51)

Reply:
Please change connection-url type to `Optional<String>` from `String`.

Reply:
Same error occurs when i make that change